src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,21 @@
 import React from 'react';
+import './CommentList.scss';
+import { Comments } from '../../types/Comment';
+import { CommentInfo } from '../CommentInfo';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  comments: Comments[]
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  <ul className="CommentList">
+    {
+      comments.map((comment) => (
+        <CommentInfo
+          key={comment.id}
+          comment={comment}
+        />
+      ))
+    }
```

Comment:
string length less than 80
```suggestion
    {comments.map((comment) => (
        <CommentInfo
          key={comment.id}
          comment={comment}
        />
      ))}
```